manifests/cluster-install/workflow-controller-rbac/workflow-controller-clusterrolebinding.yaml

```diff
@@ -9,4 +9,4 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: argo
-  namespace: default
```

Comment:
Hi @alexec!
Modifying this line break compatibility with dynamic change namespace insert by me here #5907